### DISCUSSION TITLE
Native app for cashiers (PoS people) to track paid/pending invoices

### DISCUSSION THREAD

**Discussion by ishristov** [btcpayserver/btcpayserver#3852] [Ideas / Feature Requests]
IMO quite often only the business owner has access to the btcpayserver and the cashiers or personnel (people that serve the customers and receive the payments) only have access to the checkout page.

Would it make sense to have a small native app for android/ios that lets people view the settled and processing payments. It can be useful to open that app to see whether an on-chain payment has been confirmed and at the end of the day/month the cashier can see the total amount of $ that were received with btcpay and report that to the owner. This can happen by creating a new guest user to btcpay but it might have better UX if a user just downloads an app, scan a QR code that gives read-only access to a store (only settled/processing invoices to keep it simple) and that's it.

**Comment by dennisreimann** [comment]
We hear ya and we'll have something for you :)

**Comment by pavlenex** [comment]
@ishristov ping me on Mattermost chat.btcpayserver.org if this is still something you're passionate about.
